### DISCUSSION TITLE
Allow negative

### DISCUSSION THREAD

**Discussion by kevinmelo** [dm4t2/vue-currency-input#405] [Q&A] [answered]
The option allowNegative inserted in version 1.16 was lost in next versions? (2 and 3)

**Comment by dm4t2** [comment]
The option was replaced by the `autoSign` option in v2 (see #193). In v3 the `autoSign` behavior is now active by default and no longer needs to be configured. You only need `valueRange: {min: 0}`.

**Reply by kevinmelo** [reply]
Oooh thanks, i can use the range too in V2 too? Because it's allow the user to type - and appers in the input but, after focus lost he clear the input

**Reply by dm4t2** [reply, marked as answer]
In v2 use `valueRange: {min: 0}` and `autoSign: true`.
